src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -152,11 +179,36 @@ public final void refreshItems() {
 
     }
 
+    private int getClosedIssuesCount() {
+        return (int) getElementsList().stream().filter((element) -> !element.getIssue().isOpen()).count();
+    }
```

Comment:
I was thinking whether we should move it to FilterPanel.java now, as it is where the List of GuiElements are now.